esp-hal/src/i2c/master/mod.rs

```diff
@@ -126,6 +126,7 @@ enum OpKind {
 /// I2C operation.
 ///
 /// Several operations can be combined as part of a transaction.
+#[derive(Debug, PartialEq, Eq, Hash, strum::Display)]
```

Comment:
This is an interesting precedent, as `strum` is not stable, but derive-implements a `core` trait. Is that something we are okay with?

Reply:
I guess the actual output string may change, but that by itself shouldn't be a semver concern. I like the idea of less code